src/nvim/fileio.c

```diff
@@ -6915,29 +6930,27 @@ char_u *get_event_name(expand_T *xp, int idx)
 }
 
 
-/*
- * Return TRUE if autocmd is supported.
- */
-int autocmd_supported(char_u *name)
+/// Return true if autocmd is supported.
+bool autocmd_supported(char_u *name)
 {
   char_u *p;
 
   return event_name2nr(name, &p) != NUM_EVENTS;
 }
 
-/*
- * Return TRUE if an autocommand is defined for a group, event and
- * pattern:  The group can be omitted to accept any group. "event" and "pattern"
- * can be NULL to accept any event and pattern. "pattern" can be NULL to accept
- * any pattern. Buffer-local patterns <buffer> or <buffer=N> are accepted.
- * Used for:
- *	exists("#Group") or
- *	exists("#Group#Event") or
- *	exists("#Group#Event#pat") or
- *	exists("#Event") or
- *	exists("#Event#pat")
- */
-int au_exists(char_u *arg)
+/// Return true if an autocommand is defined for a group, event and
+/// pattern:  The group can be omitted to accept any group.
+/// `event` and `pattern` can be NULL to accept any event and pattern.
+/// Buffer-local patterns <buffer> or <buffer=N> are accepted.
+/// Used for:
+///   exists("#Group") or
+///   exists("#Group#Event") or
+///   exists("#Group#Event#pat") or
+///   exists("#Event") or
+///   exists("#Event#pat")
+///
+/// @param arg The autocommand string
+bool au_exists(char_u *arg)
```

Comment:
Same: FUNC_ATTR_WARN_UNUSED_RESULT for function that can be considered PURE. `char_u *arg` can definitely be `const char_u *arg`.